libtiny_client/src/state.rs

```diff
@@ -491,8 +495,21 @@ impl StateInner {
 
     fn get_chan_nicks(&self, chan: &str) -> Vec<String> {
         match utils::find_idx(&self.chans, |(s, _)| s == chan) {
-            None => vec![], // TODO: Log this, this is probably a bug
-            Some(chan_idx) => self.chans[chan_idx].1.iter().cloned().collect(),
+            None => {
+                error!("Could not find channel index in get_chan_nicks.");
+                vec![]
+            }
+            Some(chan_idx) => {
```

Comment:
Sorting here now to leave the logic out of the cmd.rs module.
Using unstable because it should be faster and we don't need to worry about equal elements (when it becomes unstable)